### PR TITLE
Expose TxnStore from m3cluster client

### PR DESCRIPTION
**Pull request by cw9**
@xichen2020

### CODE REVIEW

File: client/etcd/client.go [hunk 1]
```diff
@@ -98,7 +98,15 @@ func (c *csclient) Services() (services.Services, error) {
 
 func (c *csclient) KV() (kv.Store, error) {
 	c.kvOnce.Do(func() {
```
Comment (xichen2020) [inline client/etcd/client.go]: Might be better to move the logic inside `KV` and `Txn` into an unexported internal function?

File: client/client.go [hunk 2]
```diff
@@ -33,4 +33,7 @@ type Client interface {
 
 	// KV returns access to the distributed configuration store
 	KV() (kv.Store, error)
+
+	// Txn returns access to the transaction store
+	Txn() (kv.TxnStore, error)
```
Comment (xichen2020) [inline client/client.go]: Would it better to call it `TxnStore`? `Txn` feels like it should return a `transaction` object

File: client/etcd/client.go (hunk 1 above)
Comment (cw9) [inline client/etcd/client.go]: sounds good

File: client/client.go (hunk 2 above)
Comment (cw9) [inline client/client.go]: sounds good

**Pull request merged**